client/src/components/MessageTextBox/MessageTextBox.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { TextField, Button } from '@mui/material';
+import axios from 'axios'
+import { Redirect } from 'react-router-dom'
+
+const MessageTextBox = (props) => {
+  const [messageText, setMessageText] = useState('')
+  const [isMessage, setIsMessage] = useState(false)
+  const [isRedirect, setIsRedirect] = useState(false)
```

Comment:
You can also remove this state, and just redirect after the message was sent successfully.